### ISSUE TITLE
[Guestbook] 2021 status of Lambdarest, call to all users!

### ISSUE THREAD

**Issue by sloev** [sloev/python-lambdarest#76]
Hi all Lambdarest users, i hope you got through 2020 alive and that 2021 will be a great year for you all.

Its 2021 and i have officially not used Lambdarest in production for nearly 3 years, so as maintainer of this project it means a lot with user feedback to keep having a sense of meaning maintaining it, so here goes... ;-)

If you have the time i would really like you all to tell me about:
1. Some stories about how you use Lambdarest
2. What problems Lambdarest has solved for you
3. What features you miss in Lambdarest
4. What you think of the quality of the repo (code, docs, test etc)
5. Are you satisfied with how the repo is being maintained? interactions like issues, PR's etc
6. What is the future for Lambdarest? still has a place?

Thanks for taking the time, and i wish you all a great year and good health 🤗 

> ❗ if you wanna support the development of this library you can do so here [buymeacoffee.com/sloev](https://buymeacoffee.com/sloev)

@andyfase, @amacks, @jacksgt, @mkreg, @aphexer, @nabrosimoff, @elviejokike, @eduardomourar, @devgro, @svdgraaf, @simongarnier, @adamelmore

**Comment by andyfase** [comment]
Hi  @sloev 

For me this library is very useful for defining RESTful API's in Lambda functions behind either API GW or Application Load Balancers. This is my primary use case where I look for a small library to provide value, without bloat and with ease of me being able to understand its full functionality.

Re other questions I like the quality of the code and am very happy with the responsiveness of accepting PR's etc. For me this repo totally still has a place. Cheers!

**Comment by mkreg** [comment]
Hi @sloev,

I found Lambdarest when looking for a simple way to create a RESTful API
using Lambda and API Gateway. It worked well and served its purpose. I
appreciated that it's small and simple. When I found some (minor) issues
with the library, I was able to understand the code easily and proposed PRs
were accepted exceptionally fast. I was definitely happy with choosing
Lambdarest for my project.

If I ever need to create a similar project I will surely try to use
Lambdarest again. I hope it will still be there to be used :)

Thanks!
